utilities/cosmos.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# set -e
+
+# force delete image and container of MS AZ Cosmos DB Emulator
+#docker container rm -f azure-cosmosdb-linux-emulator
+#docker image rm -f mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator:latest
+
+PORT=$1
+JVM_HOME=$2
+
+# Main ...
+if [ -z "$PORT" ]
+then
+  PORT=8081
+  echo "CosmosDB starting on DEFAULT port $PORT"
+else
+  echo "CosmosDB starting on specific port $PORT"
+fi
+
+if [ -z "$JVM_HOME" ]
+then
+  JVM_HOME=$JAVA_HOME
+  echo "Using DEFAULT JAVA_HOME $JVM_HOME"
+else
+  echo "Using custom JAVA_HOME $JVM_HOME"
+fi
+
+
+# Azure Cosmos DB Emulator
+URL="https://localhost:$PORT/_explorer/index.html"
+
+ipaddr=$(ifconfig | grep "inet " | grep -Fv 127.0.0.1 | awk '{print $2}' | head -n 1)
+echo "Using ${ipaddr} for CosmosDB configuration..."
+
+docker pull mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator
+
+docker run \
+    --detach \
+    --publish $PORT:8081 \
```

Comment:
`$PORT` cannot be changed, Cosmos have a bug, so it should be `8081`
👀 https://github.com/Azure/azure-cosmos-db-emulator-docker/issues/53

Reply:
removed `$Port`